.eslintrc.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
```

Comment:
Why is this in a JS file now?

Reply:
Just what `npm init @eslint/config` defaults to, I think.